src/vmm/src/version_map.rs

```diff
@@ -17,9 +17,9 @@ lazy_static! {
 
     /// Static instance used for mapping Firecracker release version to
     /// snapshot data format version.
-    pub static ref FC_VERSION_TO_SNAP_VERSION: HashMap<u16, u16> = {
+    pub static ref FC_VERSION_TO_SNAP_VERSION: HashMap<String, u16> = {
         let mut hm = HashMap::new();
```

Comment:
nit (and unrelated to the new changes):

`hm` is not a very helpful variable name. It would be awesome if we could find a more suggestive name :-?.

Reply:
I tried 😅